_stbt/control.py

```diff
@@ -18,20 +20,21 @@
 
 def uri_to_remote(uri, display=None):
     remotes = [
-        (r'none', NullRemote),
-        (r'test', lambda: VideoTestSrcControl(display)),
-        (r'vr:(?P<hostname>[^:/]+)(:(?P<port>\d+))?', VirtualRemote),
-        (r'samsung:(?P<hostname>[^:/]+)(:(?P<port>\d+))?',
-         _new_samsung_tcp_remote),
-        (r'lirc(:(?P<hostname>[^:/]+))?:(?P<port>\d+):(?P<control_name>.+)',
-         new_tcp_lirc_remote),
-        (r'lirc:(?P<lircd_socket>[^:]+)?:(?P<control_name>.+)',
-         new_local_lirc_remote),
         (r'''irnetbox:
              (?P<hostname>[^:]+)
              (:(?P<port>\d+))?
              :(?P<output>\d+)
              :(?P<config>[^:]+)''', IRNetBoxRemote),
+        (r'lirc(:(?P<hostname>[^:/]+))?:(?P<port>\d+):(?P<control_name>.+)',
+         new_tcp_lirc_remote),
+        (r'lirc:(?P<lircd_socket>[^:]+)?:(?P<control_name>.+)',
+         new_local_lirc_remote),
+        (r'none', NullRemote),
+        (r'roku:(?P<hostname>[^:]+)', RokuHttpControl),
+        (r'samsung:(?P<hostname>[^:/]+)(:(?P<port>\d+))?',
+         _new_samsung_tcp_remote),
+        (r'test', lambda: VideoTestSrcControl(display)),
+        (r'vr:(?P<hostname>[^:/]+)(:(?P<port>\d+))?', VirtualRemote),
```

Comment:
Indeed, I made sure to leave the two `lirc` entries in the same order relative to each other.